Source/UI/CommonUI/View/Component/BorderView.swift

```diff
@@ -28,6 +28,7 @@ final class BorderView: UIView {
     /// - Parameter style: protocol with some parameters for styling border and corners
     func update(with style: ElementBorderStyle) {
         self.style = style
+        borderLayer.createCustomBorder(with: style)
     }
```

Comment:
We could be more efficient here and only createCustomBorder if style is different from previous style. We might also need to remove an existing style before adding another.

Reply:
What I am saying is that if the same style is assigned, we can save some processing by not having to create a new path and assign it. I cannot see anything at code level being efficient on this, so currently if a style is provided, the path is created **every time**